spec/dummy/config/environment.rb

```diff
@@ -0,0 +1,7 @@
+# Load the Rails application.
+require_relative 'application'
+
+load File.expand_path('../../../lib/arel_toolkit.rb', __dir__)
```

Comment:
Instead of requiring, I'm (re)loading the arel_toolkit here to allow railtie to load.